python/storage-cleaner/src/utils.py

```
@@ -0,0 +1,40 @@
+    
+    :param obj: Dictionary to check
+    :param keys: List of required keys
+    :raises ValueError: If required keys are missing
+    """
+    missing = []
+    for key in keys:
+        if key not in obj or obj[key] is None or obj[key] == 0:
+            missing.append(key)
+
+    if missing:
+        raise ValueError(f"Missing required fields: {', '.join(missing)}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)